build.gradle

```diff
@@ -11,11 +11,12 @@ import com.synopsys.integration.log.PrintStreamIntLogger
 buildscript {
     ext {
         springBootVersion = '2.1.5.RELEASE'
-        blackDuckCommonVersion = '44.4.5'
+        blackDuckCommonVersion = '44.5.0-SNAPSHOT'
         polarisCommonVersion = '0.13.2'
         junitPlatformDefaultTestTags = "integration, performance, battery"
     }
     repositories {
+        mavenLocal() // TODO: Remove me after releasing integration-bdio and blackduck-common
```

Comment:
I don't think this should have been checked in. Shouldn't we just wait until 44.5.0 is released to merge?

Reply:
I created the PR to be reviewed in conjunction with the balckduck common changes. If there are any objections I would like to hear them before release